### PR TITLE
Add network tag to place page

### PR DESCRIPTION
**Pull request by dvdmrtnz**
This PR adds the network tag to the place page and allows more POIs to show the operator tag.

* Fixes https://github.com/organicmaps/organicmaps/issues/4705
* Fixes https://github.com/organicmaps/organicmaps/issues/6657

Icon from https://github.com/FortAwesome/Font-Awesome/blob/6.x/svgs/solid/circle-nodes.svg

Examples:

### CODE REVIEW

**Comment by biodranik** [comment]
How to explain what is "network" and what does this icon mean to users?

**Comment by biodranik** [comment]
Does it make sense to add "Network: ..." and "Operator: ..." in the same way as it is written "Capacity: xxx"?

File: generator/osm2meta.cpp [hunk 1]
```diff
@@ -102,22 +102,7 @@ std::string MetadataTagProcessorImpl::ValidateAndFormat_stars(std::string const
...
-      ftypes::IsCarSharingChecker::Instance()(t) ||
-      ftypes::IsCarRentalChecker::Instance()(t) ||
-      ftypes::IsBicycleRentalChecker::Instance()(t) ||
-      ftypes::IsParkingChecker::Instance()(t))
-  {
-    return v;
-  }
-
-  return {};
+  return v;
```
Comment (vng) [inline generator/osm2meta.cpp:105]: I have doubts here. There will be too many additional _useless_ (from my perspective) strings. Who cares about "operator" for power=* ? The same is for "network". Who cares about the highway's network? I think we should keep the existing amenity (POI) checker. Ok, probably add PT.

https://taginfo.openstreetmap.org/keys/operator#combinations
https://taginfo.openstreetmap.org/keys/network#combinations

**Review by vng — CHANGES_REQUESTED** [review]
Should we also index "network" for search like "operator"?

File: generator/osm2meta.cpp (hunk 1 above)
Comment (dvdmrtnz) [reply generator/osm2meta.cpp]: I reverted back to the previous but added some more tags to operator validator

**Comment by dvdmrtnz** [comment]
> Does it make sense to add "Network: ..." and "Operator: ..." in the same way as it is written "Capacity: xxx"?

Added

**Comment by dvdmrtnz** [comment]
> Should we also index "network" for search like "operator"?

Where is "operator" added to search index in code?

**Comment by vng** [comment]
I did a PR with IsPoiChecker https://github.com/organicmaps/organicmaps/pull/8595.
The changes are major, so better to merge and update after the upcoming release.

**Comment by pastk** [comment]
> The operator tag is translated as an "Owner" which is wrong - there is a separate owner tag for that.

(first point in https://github.com/organicmaps/organicmaps/issues/1948, ref [osm wiki](https://wiki.openstreetmap.org/wiki/Key:operator#Owner_or_brand_name_isn't_necessarily_the_operator))

Also seeing both "Operator" and "Network" (which are different) seems a bit confusing to me. What info do they convey to an end-user?


Operator:
> The operator tag is used to name a company, corporation, person or any other entity who is directly in charge of the current operation of a map object. 

(e.g. a franchise restaurant could be operated by some small / unknown company)

Network:
> In conjunction with [amenity](https://wiki.openstreetmap.org/wiki/Key:amenity)=[bicycle_rental](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbicycle_rental) or [amenity](https://wiki.openstreetmap.org/wiki/Key:amenity)=[atm](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Datm) or [amenity](https://wiki.openstreetmap.org/wiki/Key:amenity)=[charging_station](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcharging_station), this key identifies an organization or brand that operates a similar amenity in various other locations. For example, a bicycle rental station in London may belong to the "tfl_cycle_hire" network. An ATM may belong to a network shared among many banks. Or a car sharing station may be part of a network where different operators allow customers to use any station in the network.

**Comment by biodranik** [comment]
I agree that showing raw types/translations from OSM is very confusing for most users. What about a very targeted approach, where only specific types with specific visual translations will be added? The choice of "owner" was motivated by this point of view.

What is the value for the end user? Is it clear to him/her? These are the main questions.

**Comment by pastk** [comment]
Yeap maybe a reason there was a limited hand-picked list of feature types for which we took "operator" is that its highly likely that operator == owner (or == brand) for those types. See https://github.com/organicmaps/organicmaps/pull/8595#discussion_r1659947484

**Comment by pastk** [comment]
I.e. in general an "operator" looks like a very niche information to be displayed.

Users are mostly interested in "brands" (chain names, etc).

**Comment by pastk** [comment]
E.g.

```
name = Burger King
operator = Restaurant Brands International
```
https://www.openstreetmap.org/node/4246568489

```
name = Burger King
operator = ULM
```
https://www.openstreetmap.org/node/2300171837

**Comment by biodranik** [comment]
@pastk your examples do not look useful for users. Who cares about some unknown operator companies?

**Comment by pastk** [comment]
Sometimes they are not good even for fuel stations:
https://www.openstreetmap.org/way/63848074
image



Maybe "operator" should be ignored if there is a "brand" or "network"?

**Comment by dvdmrtnz** [comment]
This PR no longer changes what types show the operator tag. This PR adds showing the network tag for the same types that currently show the operator, and also adds "Operator:" and "Network:" prefixes before the operator/network name.

**Comment by Jean-BaptisteC** [comment]
Can you rebase your branch?

**Comment by biodranik** [comment]
What is the useful list of OSM types where the displaying of the "network" value brings real benefits to users?

I don't have a clear answer on how to avoid confusion in the UI.

**Comment by dvdmrtnz** [comment]
> What is the useful list of OSM types where the displaying of the "network" value brings real benefits to users?

In ATMs usually you would have to pay more fees if it's not the same network as your bank.
In Car Sharing/Bike Rentals/Car Rentals you have to drop off your car/bike at a station of the same network.
In Electric Car Charging Stations you may be already subscribed/get discounts? on a specific network?

**Pull request merged**